### PR TITLE
Add private field to MessageEntity

### LINKED ISSUE #19: Private/public message  Edit messages
1. Should we not have an option for Private/Public messages

2. Who is supposed to Edit messages, Titles etc... Is it just the User that created the message or do we want Admin to have the option aswell

### LINKED ISSUE #23: Add springboot validator Dependency and add validators to variables in Dto record
Suggested changes

1. Add Spring-boot-starter-validation dependency 

2. Make Dto records to class and add validations for the variables that are being used

### PR DESCRIPTION
**Pull request by ValentinaSukonina**
### Related Issue(s)
Closes #19 
Closes #23 
Closes #24 

### Description
Field private is added to MessageEntity to allow sorting of messages depending on this status

Corresponding field is removed from MessageDto, and all classes are updated accordingly, including tests

### CODE REVIEW

File: src/main/java/dev/ebrydeu/spring_boot_library/controllers/MessageController.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/messages")
+@RequiredArgsConstructor
+public class MessageController{
+    private final MessageService messageService;
+
+    @PutMapping("/{id}/private/{messagePrivate}")
```
Comment (Ebyrdeu) [inline src/main/java/dev/ebrydeu/spring_boot_library/controllers/MessageController.java:17]:
```suggestion
    @PatchMapping("/{id}/private/{messagePrivate}")
```
You don't need full update for one field just use partial

**Review by Ebyrdeu — CHANGES_REQUESTED** [review]

**Review by ajohnsson95 — CHANGES_REQUESTED** [review]
Looks great, small change that maxim mentioned but good job :)

**Review by Ebyrdeu — APPROVED** [review]

**Review by ajohnsson95 — APPROVED** [review]

**Review by ajohnsson95 — APPROVED** [review]

**Comment by Ebyrdeu** [comment]
Just to be sure, this PR resolve both  #23 and #19  ?

**Comment by ajohnsson95** [comment]
> Just to be sure, this PR resolve both #23 and #19 ?

Yes correct

**Review by Ebyrdeu — APPROVED** [review]

**Pull request merged**